src/launch.ts

```diff
@@ -157,9 +179,9 @@ export class Launcher implements vscode.Disposable {
             type: dbg,
             name: `Debug My Program`,
             request: 'launch',
-            cwd: '${command:makefile.launchTargetDirectory}',
+            cwd: '${command:makefile.getLaunchTargetDirectory}',
             args,
-            program: '${command:makefile.launchTargetPath}',
+            program: '${command:makefile.getLaunchTargetPah}',
```

Comment:
Would we rather have it build before debugging by default?

Reply:
This is the internal launch functionality and there is already a build before this point, also controlled by the same setting.

Reply:
So since this is the just the internal launch functionality, should we call `getLaunchTargetPath()` ourselves and put the expected string in here instead of sending a command variable which will result in another callback to resolve it?

Reply:
Oh interesting, I didn't realize that. Good point.